Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs

```diff
@@ -215,6 +231,15 @@ private void HandleTestCompletion(object sender, TestCompletedEventArgs e)
             Tests.Refresh();
         }
 
+        public IRewritingManager RewritingManager { get; }
+
+        public string UnignoreTestLabel => RubberduckUI.TestExplorer_TestToggle_Unignore;
+        public string IgnoreTestLabel => RubberduckUI.TestExplorer_TestToggle_Ignore;
```

Comment:
AFAICT these should be retrieved in the XAML and not exposed in the ViewModel. At least the other MenuItems in the TestExplorer are using `{Resx [...]}` instead of `{Binding [...]}`